src/core_plugins/console/public/index.html

```diff
@@ -12,11 +12,20 @@
           </a>
         </kbn-tooltip>
         <span dropdown>
-          <a class="editor_action" dropdown-toggle href="#">
-            <i class="fa fa-wrench"></i>
+          <a
+            id="consoleRequestOptions"
```

Comment:
Want me to add a note to the [HTML style guide](https://github.com/elastic/kibana/blob/master/style_guides/html_style_guide.md)?